single-line-diagram-core/src/main/java/com/powsybl/sld/layout/AbstractLayout.java

```diff
@@ -42,6 +44,12 @@ protected void manageSnakeLines(BaseGraph graph, LayoutParameters layoutParamete
                     ((BranchEdge) adjacentEdges.get(i)).setSnakeLine(pollingSplit.get(i));
                 }
             }
+
+            if (multiNode.getAdjacentEdges().size() == 2) {  // 2 windings transformer
+                handle2wtNodeRotation(multiNode);
+            } else {  // 3 windings transformer
+                handle3wtNodeRotation(multiNode);
+            }
```

Comment:
I think you can put this two in the for loop above, to avoid the if again, and put the `List<List<Point>>` as argument instead of retrieving it